src/00b3_survey_mics/script.R

```diff
@@ -171,29 +237,42 @@ labels_dfs <- lapply(labels, function(x) {
 circ_extracted <- lapply(seq_along(circ_extracted), function(i) {
   if ("mics_area_name" %in% names(circ_extracted[[i]])) {
     left_join(
-      circ_extracted[[i]], labels_dfs[[i]], by = "mics_area_name" 
-    ) %>% 
+      circ_extracted[[i]], labels_dfs[[i]], by = "mics_area_name"
+    ) %>%
       select(-matches("mics_area_name"))
   } else return(circ_extracted[[i]])
   left_join(
-    circ_extracted[[i]], labels_dfs[[i]], by = "mics_area_name" 
-  ) %>% 
+    circ_extracted[[i]], labels_dfs[[i]], by = "mics_area_name"
+  ) %>%
     select(-matches("mics_area_name"))
 })
 names(circ_extracted) <- names(circ_raw)
 
 # OLI: Note on the value_recode tab of the excel file
-# There are several cases where though the var name is custom to the survey, 
-# the value coding is the same as the default. 
-# The value recode entries for those surveys can be removed, but for speed I 
-# just added them all. I'm not sure removing them is any better than leaving 
-# them in though. The size of the value recode book is immaterial. 
+# There are several cases where though the var name is custom to the survey,
+# the value coding is the same as the default.
+# The value recode entries for those surveys can be removed, but for speed I
+# just added them all. I'm not sure removing them is any better than leaving
+# them in though. The size of the value recode book is immaterial.
+
+#' JE edit: add value recode for CAF2006MICS in mm dataset
+
+value_recode <- value_recode %>%
+  bind_rows(
+    tribble(
+      ~survey_id,    ~dataset, ~variable,     ~val_raw, ~val_label_raw, ~value, ~notes, ~var_raw, ~analysis,
+      "CAF2006MICS", "mm",     "circ_status", "1",      "Oui",          "1",    NA,     "hha19",  "circ",
+      "CAF2006MICS", "mm",     "circ_status", "2",      "Non",          "0",    NA,     "hha19",  "circ",
+      "CAF2006MICS", "mm",     "circ_status", "9",      "Manquant",     NA,     NA,     "hha19",  "circ"
```

Comment:
I think this needed a custom recode specified because the `dataset` is `mm` even though the recode is the same.  A bit of a pinch point but not terrible if that's the best option.

(This is ultimately not used because I dropped the `CAF2006MICS`, but kept the code here for future use if needed)